inc/Engine/Preload/preload.php

```diff
@@ -2,6 +2,10 @@
 
```

Comment:
This file should be in inc/API/preload.php instead I believe

Reply:
Resolved here https://github.com/wp-media/wp-rocket/pull/2396/commits/8675a6bc19165f5173f4f6d11bbd6e13bcfc005c